runtime/src/main/kotlin/kotlin/collections/Collections.kt

```diff
@@ -41,6 +43,23 @@ public interface MutableIterable<out T> : Iterable<T> {
 }
 
 
+@PublishedApi
+@SinceKotlin("1.3")
+@ExperimentalStdlibApi
+@kotlin.internal.InlineOnly
+internal actual inline fun <E> buildListInternal(@BuilderInference builderAction: MutableList<E>.() -> Unit): List<E> {
```

Comment:
Since these functions are internal they likely do not need `@BuilderInference` annotation.

Reply:
Fixed